src/engine/execute.js

```diff
@@ -191,6 +191,18 @@ class BlockCached {
          */
         this.mutation = cached.mutation;
 
+        /**
+         * Profiler block is configured with.
```

Comment:
I read this wrong and thought it was an incomplete sentence at first.
```suggestion
         * The profiler the block is configured with.
```

Reply:
Added.